recipes-support/blink-led/blink-led/blink-led

```diff
@@ -20,3 +20,12 @@ elif rauc status | grep -q 'Booted from: rootfs.0 (system0)' &&
     echo 0 > /sys/class/leds/led-red/brightness
     echo heartbeat > /sys/class/leds/led-green/trigger
 fi
+rauc_status_old=$(rauc status)
+
+while true; do
+	if [ "$rauc_status_old" != "$(rauc status)" ]
```

Comment:
Please check the indentation here. Seems to be off (using tabs instead of spaces?)